doc/cephfs/mount-using-kernel-driver.rst

```diff
@@ -50,51 +50,47 @@ Synopsis
 ========
 In general, the command to mount CephFS via kernel driver looks like this::
 
-    mount -t ceph {device-string}:{path-to-mounted} {mount-point} -o {key-value-args} {other-args}
+    mount -t ceph {device-string}={path-to-mounted} {mount-point} -o {key-value-args} {other-args}
 
 Mounting CephFS
 ===============
 On Ceph clusters, CephX is enabled by default. Use ``mount`` command to
 mount CephFS with the kernel driver::
 
     mkdir /mnt/mycephfs
-    mount -t ceph :/ /mnt/mycephfs -o name=foo
+    mount -t ceph <name>@<fs_name>[.<fsid>]=/ /mnt/mycephfs -o mon_host=<mon_host>
```

Comment:
After reading your comment

https://github.com/ceph/ceph/pull/41334#issuecomment-841152689

I'm thinking we might change to `name@fsid.<fs_name>`. I think one of the lessons with DNS is that specificity should go from left to right. Similar to Java: `org.foo.bar`. This is a good excuse to deftly avoid the ambiguity of periods in the `fs_name`.

Reply:
Also another thought: we should think about the arguments at the syscall level versus the shell level. At the system call level, putting the fsid makes sense. We want the kernel to use that. But at the shell level (i.e. what's passed to `mount.ceph`) it would be more use-friendly and consistent to take a cluster name. That would cause `mount.ceph` to lookup the `ceph.conf` corresponding to that cluster, get the fsid, and pass that on to the kernel. Makes sense?

Reply:
Right, the fsid there is kind of making the syntax a bit awful (especially when compared to elegant `mon_ip:/path`). Let's make the fsid a mount param (optional, since it can be picked up from ceph.conf) -- that makes device string parsing much easier?